src/args.rs

```diff
@@ -148,8 +151,31 @@ pub fn parse() -> ArgMatches<'static> {
                         .help("Watch specific file(s) or folder(s)"),
                 )
                 .after_help(footnote.as_str()),
-        )
-        .get_matches();
+        );
+
+    // Allow invocation of cargo-watch with both `cargo-watch watch ARGS` (as invoked by cargo) and `cargo-watch ARGS`.
+    let mut args: VecDeque<String> = env::args().collect();
```

Comment:
Not much point using a VecDeque instead of a Vec here now.

Reply:
Fixed.